src/main/resources/page/js/dummy-endpoint-caller.js

```diff
@@ -0,0 +1,8 @@
+class APICaller {
+    constructor() {}
+
+    static async getDateAndTimeInJson(url) {
```

Comment:
Should the URL belong to the class itself?

Reply:
No, it is from a parameter for we could call this whenever we want data from an API. I will rename the function and I think it will be clearer.